packages/blockchain-environment-v1/src/FabricEnvironmentConnection.ts

```diff
@@ -500,4 +498,30 @@ export class FabricEnvironmentConnection implements IFabricEnvironmentConnection
 
         return peer.getAllChannelNames();
     }
+
+    private async instantiateOrUpgradeChaincode(_name: string, _version: string, _peerNames: Array<string>, _channelName: string, _fcn: string, _args: Array<string>, _collectionPath: string, _contractEP: any, isUpgrade: boolean): Promise<any> { // FIXME was buffer
```

Comment:
`instantiateOrUpgradeSmartContractDefinition` seems to return `void` (is this correct?), so we should make this `Promise<void>`?

Same for `instantiateChaincode`